clowder-framework/clowder
# Upgrade jsonform to version 2.2.5

## Description

### Problem
Our version of JSONForm  is a bit outdated, and does not work for all examples provided in the library's documentation.

### Approach
* Upgraded JSONForm to version 2.2.5

### How to Test
1. Following the example [here](https://jsonform.github.io/jsonform/playground/index.html?example=navigation-tabs), build up an `extractor_info.json` containing these parameters
2. Register the extractor with Clowder
3. Navigate to any file's "Submit to Extractor" view
    * You should see your new extractor listed here
    * Your should see the extractor's parameter list offers a rich/complex form including navigational tabs

## Review Time Estimate


## Types of changes


## Checklist:
